server/templates/StudyDetails.template.js

```diff
@@ -0,0 +1,26 @@
+import basePathConfig from '../configs/basePathConfig'
+const serialize = require('serialize-javascript');
+
+const basePath = basePathConfig || ''
+
+export default (user) => {
+  return `
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
+        <link rel='stylesheet' href='${basePath}/css/study_details.css' />
+        <title>Study Details - DPDash</title>
+        <script>
+          window.USER = ${serialize(user, {isJSON: true})}
+          window.SUBJECT = ${serialize(null, {isJSON: true})}
+          window.GRAPH = ${serialize(null, {isJSON: true})}
+        </script>
```

Comment:
I dislike having to add this and would like to away with them, as it turns out we have to set these values, because almost every component uses redux and initial values for redux are malfunctioning. Could we make a card to fix the initial values for these redux values? 
We could even do away with redux it seems that it's required because of @material-ui and how the styles are being fed through each page and component. 

https://github.com/AMP-SCZ/dpdash/blob/19e87a7067e6d9ab1b385d9f8a27b9bd00f5f858/stores/reducers.js#L4-L16

The sidebar is hooked up to Redux
https://github.com/AMP-SCZ/dpdash/blob/19e87a7067e6d9ab1b385d9f8a27b9bd00f5f858/views/components/Sidebar.jsx#L78-L80